src/App.tsx

```diff
@@ -1,10 +1,43 @@
 import './App.scss';
-import { Slider } from './components/Slider';
+
 import { BasicCard } from './components/Slider/SliderExample';
+import { Slider } from './components/Slider';
+import { useEffect } from 'react';
+import { useSanity } from './hooks/useSanity';
 
 function App() {
+  const sanity = useSanity();
+
+  useEffect(() => {
+    sanity.changeLanguage(window.location.pathname.split('/').slice(-1)[0]);
+  }, [sanity]);
+
+  //only the above will be merged
+
+  const bts = ['en', 'pl', 'de'].map(l => (
```

Comment:
Tutaj jezeli chodzi o te tablice z jezykami to wolałbym aby pochodziła ona z jednego miejsca. Czyli np z jakiegos globalnego obiektu w folderze Constants czy cos takiego